disnake/state.py

```diff
@@ -1268,6 +1270,60 @@ def parse_guild_stickers_update(self, data) -> None:
         guild.stickers = tuple(map(lambda d: self.store_sticker(guild, d), data["stickers"]))  # type: ignore
         self.dispatch("guild_stickers_update", guild, before_stickers, guild.stickers)
 
+    def parse_guild_scheduled_event_create(self, data) -> None:
+        scheduled_event = GuildScheduledEvent(state=self, data=data)
+        guild = scheduled_event.guild
+        if guild is not None:
+            self._scheduled_event_guilds[scheduled_event.id] = scheduled_event.guild_id
+            guild._scheduled_events[scheduled_event.id] = scheduled_event
+        self.dispatch("guild_scheduled_event_create", scheduled_event)
```

Comment:
These events should be in the docs as well, and (if I'm guessing correctly that these need the `guilds` intent(?)) also in the `Intents.guilds` docstring